pipenv/core.py

```diff
@@ -720,7 +720,6 @@ def batch_install(
     if sequential_deps is None:
         sequential_deps = []
     failed = not retry
-    install_deps = not no_deps
```

Comment:
Essentially moving this to line 759 is what the original PR did that this replaces -- the reason why is because line 757 can override this for a specific artifact, but we should not use that flag for all iterations of the loop.